doctest/test_docs.py

```diff
@@ -48,10 +48,34 @@ def process(self, statement):
         click.echo(output)
 
 
+"""
+For _explain requests, there are several additional request fields that will inconsistently
+appear/change depending on underlying cluster state. This method normalizes these responses in-place
+to make _explain doctests more consistent.
+
+If the passed response is not an _explain response, the input is left unmodified.
+"""
+def normalize_explain_response(data):
```

Comment:
It seems all query reponse are impacted, not only the explain query. do u know why the format is different? does it impact rest api customer?

Reply:
Formatting changes because of `cli-helpers` updating from v1 to v2, which removes some extra whitespace from most responses. It's separate from the `_explain` issue.